src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -236,14 +236,32 @@ protected Task Stop()
             RequestContext<object> requestContext)
         {
             if (helpParams == null) { helpParams = "get-help"; }
-
-            var psCommand = new PSCommand();
-            psCommand.AddCommand("Get-Help");
-            psCommand.AddArgument(helpParams);
-            psCommand.AddParameter("Online");
-
-            await editorSession.PowerShellContext.ExecuteCommand<object>(psCommand);
-
+            string script = @"
+function __Check-Help {
+    param (
+        $Command
+    )
+        $CommandExists = $false
+        Try {
+            Get-Command $Command -ErrorAction Stop
+            $CommandExists = $true
+        } catch [System.Management.Automation.CommandNotFoundException] {
+            Write-Warning ""$Command does not exist.""
+        }
+        if ($CommandExists) {
+            try {
+                Get-Help $Command -Online
+            } catch [System.Management.Automation.PSInvalidOperationException] {
+                Get-Help $Command -Full
+            }
+        }
+}";
+            PSCommand checkHelpScript = new PSCommand();
+            checkHelpScript.AddScript(script);
```

Comment:
After removing the `function` part of the script, change this to add the argument here instead and remove the second command invocation.  Also add `useLocalScope: true` to `AddScript` (so the variables don't leak into global)